storage/redis/client_test.go

```diff
@@ -181,7 +181,7 @@ func TestCrudValidConnection(t *testing.T) {
 				assert.NoError(t, err)
 
 				value, err := st.Get(storage.Key(unexistingKey))
-				assert.NoError(t, err)
+				assert.Error(t, err)
```

Comment:
here you would do assert.True(t, ErrKeyNotFound.Has(err)) or something

Reply:
thanks!